docs/troubleshooting-guide/loading-from-h2.md

```diff
@@ -0,0 +1,33 @@
+## Loading exported application database fails 
+
+If you've been using the default H2 application database that ships with Metabase, and want to [migrate from the default H2 application database][migrate] to a production database like [PostgreSQL][postgres], you'll need to use the `load-from-h2` command, which will fail if the database filename is incorrect. 
```

Comment:
"like Postgres or MySQL" (to be consistent with what's above)?

Reply:
Also, is it worth saying that `load-from-h2` is built into Metabase? My first thought was, "Where's that executable stored on disk and how do I add it to my path?"

Reply:
We should probably consistently write "Postgres or MySQL/MariaDB"